tbx/static_src/sass/components/_icon.scss

```diff
@@ -24,6 +24,11 @@
         width: 100%;
     }
 
+    &--listing-arrow {
+        @include arrow-link-icon-styles();
+        margin-left: $spacer-mini-plus;
```

Comment:
Without this the arrow on titles and arrows that are only on one line overlap the focus ring:

<details><summary>Details</summary>
<p>


![image](https://github.com/user-attachments/assets/00031ac1-47df-493a-8a8a-98a5d14badaa)

</p>
</details>

Reply:
nitpick: maybe add an explanatory comment for why we need the margin.

Reply:
Added 👍